### PR TITLE
updated to v2.6.0

### PR DESCRIPTION
**Pull request by deolank**
*Issue #, if available:*

*Description of changes:*

 - Updated the solution to use CDK V2 to generate CloudFormation templates and support CDK deployments.
 - Updated the 'Terminate Workspace' feature to accept user input for number of months to check for before terminating unused workspaces.
 - Added a retention policy of 365 days to ECS logs to optimize the costs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by WillAWS — APPROVED** [review]
Small url link changes. Other than that, looks good to me.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ assignees: ""
 
 - [ ] Version: [e.g. v1.0.0]
 
-To get the version of the solution, you can look at the description of the created CloudFormation stack. For example, "_(SO0009) - The AWS CloudFormation template for deployment of the aws-centralized-logging. Version v1.0.0_". You can also find the version from [releases](https://github.com/aws-solutions/aws-centralized-logging/releases)
+To get the version of the solution, you can look at the description of the created CloudFormation stack. For example, "_(SO0009) - The AWS CloudFormation template for deployment of the aws-centralized-logging. Version v1.0.0_". You can also find the version from [releases](https://github.com/aws-solutions/workspaces-cost-optimizer/releases)
```
Comment (groverlalit) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Can you please update this link as well? Thanks.

**Review by WillAWS — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (groverlalit) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done. Thanks

**Review by groverlalit — APPROVED** [review]
Looks good now.

**Pull request merged**